app/assets/stylesheets/themes/default/pages/_matches.scss

```diff
@@ -167,21 +168,8 @@ div#match {
         position: relative;
       }
 
-      .controls {
-        top: 0;
-        position: absolute;
-        z-index: 100;
-
-        input {
-          margin: 0;
-        }
-      }
-
-      form {
-
-        label.checkbox {
-          padding: 0;
-        }
+      label.checkbox {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4